dist/angular-patternfly.js

```diff
@@ -771,10 +796,10 @@ angular.module('patternfly.charts').directive('pfDonutPctChart', ["c3ChartDefaul
             centerLabelText.smText = '';
           } else if ($scope.centerLabel === 'available') {
             centerLabelText.bigText = $scope.data.available;
-            centerLabelText.smText = $scope.config.units + ' Available';
+            centerLabelText.smText = $scope.config.units + ' ' + $scope.lang.available;
           } else if ($scope.centerLabel === 'percent') {
             centerLabelText.bigText = Math.round($scope.data.used / $scope.data.total * 100.0) + '%';
-            centerLabelText.smText = 'of ' + $scope.data.total + ' ' + $scope.config.units;
+            centerLabelText.smText = $scope.lang.of + ' ' + $scope.data.total + ' ' + $scope.config.units;
```

Comment:
String concat is bad since we need to handle the ability to reorder strings in say Japanese. I need to see what we can do there but I wanted to get some feedback on this approach